src/main/java/com/theironyard/controllers/IronQuotesController.java

```diff
@@ -0,0 +1,167 @@
+import com.theironyard.services.QuoteRepository;
+import com.theironyard.services.TagRepository;
+import com.theironyard.services.UserRepository;
+import com.theironyard.utilities.PasswordStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Created by EddyJ on 8/1/16.
+ */
+@RestController
+public class IronQuotesController {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    QuoteRepository quoteRepository;
+
+    @Autowired
+    TagRepository tagRepository;
+
+    @RequestMapping(path = "/quotes", method = RequestMethod.POST)
+    public Quote createQuote(@RequestHeader(value = "Authorization") String userToken,@RequestBody Quote quote){
+        getUserFromAuth(userToken);
+        quoteRepository.save(quote);
+        return quote;
+    }
+
+    @RequestMapping(path = "/quotes",method = RequestMethod.GET)
+    public List<Quote> getAllQuotes(){
+        List<Quote> quoteList = quoteRepository.findAll();
+        return quoteList;
+    }
+
+    @RequestMapping(path = "/quotes/{id}",method = RequestMethod.GET)
+    public Quote getQuote(@PathVariable Integer id){
+        Quote quote = quoteRepository.findOne(id);
+        return quote;
+    }
+
+    @RequestMapping(path = "/quotes/{id}",method = RequestMethod.PUT)
+    public Quote updateQuote(@RequestHeader(value = "Authorization") String userToken,@PathVariable Integer id, @RequestBody QuoteCommand quoteCommand){
+        getUserFromAuth(userToken);
+        Quote quote = quoteRepository.findOne(id);
+        quote.setText(quoteCommand.getText());
+        quote.setAuthor(quoteCommand.getAuthor());
+        quoteRepository.save(quote);
+        return quote;
+    }
+
+    @RequestMapping(path = "/quotes/{id}",method = RequestMethod.DELETE)
+    public void deleteQuote(@RequestHeader(value = "Authorization") String userToken,@PathVariable Integer id) throws Exception {
+        getUserFromAuth(userToken);
+        Quote deletedQuote = quoteRepository.findOne(id);
+        if(deletedQuote == null){
+            throw new Exception("Quote does not exist");
+        }
+        quoteRepository.delete(deletedQuote);
+    }
+
+    @RequestMapping(path = "/tags",method = RequestMethod.GET)
+    public List<Tag> getAllTag(){
+        List<Tag> tagList = tagRepository.findAll();
+        return tagList;
+    }
+
+    @RequestMapping(path = "/tags/{id}",method = RequestMethod.GET)
+    public Tag getTag(@PathVariable Integer id){
+        Tag tag = tagRepository.findOne(id);
+        return tag;
+    }
+
+    @RequestMapping(path = "/quotes/{id}/tags",method = RequestMethod.POST)
+    public Tag addTag(@RequestHeader(value = "Authorization") String userToken,@PathVariable Integer id, @RequestBody TagCommand tagcommand){
+        getUserFromAuth(userToken);
```

Comment:
This authorization doesn't do anything.  You just get the user but you don't make sure that they are authorized.